src/adhocracy_mercator/adhocracy_mercator/evolution/__init__.py

```diff
@@ -197,3 +254,4 @@ def includeme(config):  # pragma: no cover
     config.add_evolution_step(remove_mercator_workflow_assignment_sheet)
     config.add_evolution_step(make_mercator_proposals_badgeable)
     config.add_evolution_step(reindex_requested_funding)
+    config.add_evolution_step(convert_rates_to_likes)
```

Comment:
Should we migrate all rates to likes or do we have installations with "rates" instead of likes?